src/with-translations.js

```diff
@@ -2,9 +2,11 @@
  *
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
  */
 
-import React, {Component} from 'react';
+import * as React from 'react';
```

Comment:
I think we had some issues with this syntax before causing things to not build correctly. It also doesn't look like you need specific types from React, so wondering if we could revert this or explicitly import types as needed?

Reply:
Hm, interesting.  It is the [recommended approach by Flow](https://flow.org/en/docs/react/types/).

We use the type for {{ComponentType}} on [line 23](https://github.com/fusionjs/fusion-plugin-i18n-react/pull/134/files#diff-6671dfa49d21c6bd618ccc6408d585b4R23) so I will import it explicitly instead.

Reply:
But I guess we do this do this in other modules now, so maybe it's fine? /shrug.